Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -1,5 +1,5 @@
 using System;
-using MLAgentsExamples;
+using Google.Protobuf.WellKnownTypes;
```

Comment:
Why do we need this ?

Reply:
removed. no idea where that came from. i ran a reformat so maybe it added that?